test/unit.js

```diff
@@ -1,6 +1,19 @@
 var test = require('tape')
+var jsdom = require('jsdom')
+var React = require('react')
+var utils = require('react-addons-test-utils')
+require('node-jsx').install({ 'extension': '.jsx' })
+var LikeButton = require('../jsx/LikeButton.jsx')
 
-test('fill in this', function (t) {
-  t.ok(true)
+test('LikeButton', function (t) {
+  // Thanks https://github.com/jesstelford/react-testing-mocha-jsdom
+  global.document = jsdom.jsdom('<body></body>')
+  global.window = global.document.defaultView;
```

Comment:
Very minor thing, 'npm test' fails standard for the semicolon on the end of this line. Could you fix this in the PR? Thanks!

```
®[widget-awesome]$ npm test
> widget-awesome@1.0.0 test /Users/ryan/redman/applicants/jczerwinski/widget-awesome
> standard && tap test/*.js

standard: Use JavaScript Standard Style (https://github.com/feross/standard) 
  /Users/ryan/redman/applicants/jczerwinski/widget-awesome/test/unit.js:11:46: Extra semicolon.
```